### PR TITLE
Fix bugs for EditClaimCommand

### PR DESCRIPTION
**Pull request by rithiklenin**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditClaimCommand.java [hunk 1]
```diff
@@ -37,12 +37,13 @@ public class EditClaimCommand extends Command {
             + PREFIX_CLAIM_STATUS + "[NEW_STATUS] "
             + PREFIX_CLAIM_DESC + "[NEW_DESCRIPTION]\n"
```
Comment (ckaayy) [inline src/main/java/seedu/address/logic/commands/EditClaimCommand.java:38]: should put square brackets around the prefix as well

**Review by ckaayy — COMMENTED** [review]
good job fixing the other bugs, just one minor one left

File: src/main/java/seedu/address/logic/commands/EditClaimCommand.java (hunk 1 above)
Comment (rithiklenin) [reply src/main/java/seedu/address/logic/commands/EditClaimCommand.java]: updated

File: src/main/java/seedu/address/logic/commands/EditPolicyCommand.java [hunk 2]
```diff
@@ -30,7 +30,7 @@ public class EditPolicyCommand extends Command {
             + ": Updates the specified policy for the client identified "
             + "by the index number used in the last client listing. \n"
             + "Parameters: INDEX (must be a positive integer) "
-            + "pt/[POLICY_TYPE] pa/[PREMIUM_AMOUNT] ca/[COVERAGE_AMOUNT] ed/[EXPIRY_DATE]\n"
+            + "pt/POLICY_TYPE [pa/PREMIUM_AMOUNT] [ca/COVERAGE_AMOUNT] [ed/EXPIRY_DATE]\n"
```
Comment (ckaayy) [inline src/main/java/seedu/address/logic/commands/EditPolicyCommand.java:33]: we should use the statically defined prefix here as well as it makes the code for adaptable to changes. (eg. if we change the statically defined prefix in the future, this wont be affected)

Comment (rithiklenin) [reply src/main/java/seedu/address/logic/commands/EditPolicyCommand.java]: Updated again!

**Review by ckaayy — APPROVED** [review]
lgtm!

**Pull request merged**